druid/src/widget/scroll.rs

```diff
@@ -390,10 +380,26 @@ impl<T: Data, W: Widget<T>> Widget<T> for Scroll<T, W> {
         }
     }
 
-    fn update(&mut self, ctx: &mut UpdateCtx, _old_data: Option<&T>, data: &T, env: &Env) {
+    fn update(&mut self, ctx: &mut UpdateCtx, _old_data: &T, data: &T, env: &Env) {
         self.child.update(ctx, data, env);
     }
 
+    fn lifecycle(&mut self, ctx: &mut LifeCycleCtx, event: &LifeCycle, data: &T, env: &Env) {
+        // The scroll bars will fade immediately if there's some other widget requesting animation.
```

Comment:
I'd reword this, "Guard against the timer id, otherwise the scroll bars would fade immediately if there's some other widget that's animating."